docs/package/gluon-hoodselector.rst

```diff
@@ -0,0 +1,130 @@
+in combination with the domain configuration system. Below you can see a visual
+example of regional domain:
+
+.. image:: hoodmap.jpeg
+
+Behavior
+------------------
+
+The following is an abstract state diagramm which gives an overview
+of the process:
+
+.. image:: gluon-hoodselector.svg
+
+The sequence of this diagram reflects the priority of its running modes.
+Each mode will be explained separately below.
+
+geolocation mode
+^^^^^^^^^^^^^^^^
+
+This mode will only be entered when a node has location coordinates set.
+Nodes with a position will set their domain based on it.
+The node will skip to the next mode when the node (a) has no position or
+(b) its position is not within any of the defined bounding boxes.
+
+default domain mode
+^^^^^^^^^^^^^^^^^^^
+
+This mode will be entered if no other modes before fits.
+It provides a fallback to the default domain.
+
+Domain shapes
+-------------
+
+There are two types of domains: one without any defined shapes
+which has to be unique and others which contain shapes.
+
+* **default domain**
+
+The default domain doesn’t hold any shapes and represents the inverted area of
+all other shapes held by other domains with geo coordinates. It will only be
+entered if a node could not be matched to a geo domain. A suggested approach is
+to define the "old" network as default domain and gradually migrate nodes from
+there to shape defined domains ("geo domain").
+
+* **geo domains**
+
+A geo domain contains shapes, which are described by three dimensional arrays
+and represents the geographical size of the domain. There are two possible
+definitions of these shapes. The first one is using rectangulars so that only
+two coordinates per box are needed to reconstruct it (see below for an
+example). The second one is using polygons which can have multiple edges.
+Each domain can hold multiple shapes.
+
+.. image:: rectangle-example.svg
+
+site.conf
+---------
+
+The designer of the shapes should always ensure that no overlapping polygons
+between domains will be created or else the order in the domain list will
+become relevant. If for example domain A and B have an overlap Hoodselector
+would, for that overlapping area, only ever reach domain A, but never domain B.
+Here is an example of a rectangular definition of a shape:
+Example::
+
+  hoodselector = {
+    shapes = {
+      {
+        {
+          lat = 53.128,
+          lon = 8.187
+        },
+        {
+          lat = 53.163,
+          lon = 8.216
+        },
+      },
+    },
+  },
+
+Here is an example of a triangle defined shape:
+Example::
+
+  hoodselector = {
+    shapes = {
+      {
+        {
+          lat = 53.128,
+          lon = 8.187
+        },
+        {
+          lat = 53.163,
+          lon = 8.216
+        },
+        {
+          lat = 53.196,
+          lon = 8.194
+        },
+      },
+    },
+  },
+
+This package is incompatible with :doc:`gluon-config-mode-domain-select`.
```

Comment:
Please make it compatible with the manual domain selection. It should always be an option to override the hoodselector.

Reply:
You can disable the hoodselector and set your domain manually. A UI to do this could be added in gluon-config-mode-domain-select, but IMO this is out of scope for this PR.